internal/pd/models.go

```diff
@@ -20,15 +20,54 @@
 
 package pd
 
+import (
+	"io/ioutil"
+	"path/filepath"
+
+	"github.com/mitchellh/go-homedir"
+	"gopkg.in/yaml.v3"
+)
+
 // Tip: Check https://yaml.to-go.online/ or https://mholt.github.io/json-to-go/
 // for an easy way to translate YAML or JSON files into Go struct code.
 
 // Config describes the pd tool configuration structure
 type Config struct {
 	Authtoken  string `yaml:"authtoken"`
+	OwnShift   string `yaml:"own-shift"`
 	ShiftTimes []struct {
 		Name  string `yaml:"name"`
 		Start string `yaml:"start"`
 		End   string `yaml:"end"`
 	} `yaml:"shift-times"`
 }
+
+//ChangeYAMLFile changes a specific value in the .pd.yml file
+func ChangeYAMLFile(name string, newValue string) error {
+
+	home, err := homedir.Dir()
+	if err != nil {
+		return err
+	}
+
+	data, err := ioutil.ReadFile(filepath.Join(home, ".pd.yml"))
+	if err != nil {
+		return err
+	}
```

Comment:
We do. The next commit will contain a new function for that but this part won't be changed, because the variable home is used in line 67 and therefore needed.